Tests failing:
```
FAILED tardis/workflows/tests/test_workflows.py::test_simple_tardis_workflow_against_test_tardis_full_regression - AssertionError: 
Not equal to tolerance rtol=1e-07, atol=0

Mismatched elements: 487956 / 584480 (83.5%)
First 5 mismatches are at indices:
 [2020, 11]: 3.3945662126384763e-22 (ACTUAL), 0.0 (DESIRED)
 [2021, 11]: 3.3877456693209765e-22 (ACTUAL), 0.0 (DESIRED)
 [2022, 11]: 3.3877456693209765e-22 (ACTUAL), 0.0 (DESIRED)
 [2023, 11]: 3.387351206641851e-22 (ACTUAL), 0.0 (DESIRED)
 [2024, 11]: 3.3854928335010723e-22 (ACTUAL), 0.0 (DESIRED)
Max absolute difference among violations: 1.23348514e-17
Max relative difference among violations: 8.467167
 ACTUAL: array([[0., 0., 0., ..., 0., 0., 0.],
       [0., 0., 0., ..., 0., 0., 0.],
       [0., 0., 0., ..., 0., 0., 0.],...
 DESIRED: array([[0., 0., 0., ..., 0., 0., 0.],
       [0., 0., 0., ..., 0., 0., 0.],
       [0., 0., 0., ..., 0., 0., 0.],...
= 1 failed, 1723 passed, 480 skipped, 32 xfailed, 3 xpassed, 84 warnings in 825.67s (0:13:45)
```

This is just a case of relative tolerance being unsuitable for stuff extremely close to exactly zero, though. Use absolute tolerance for this one?

--- a/tardis/workflows/tests/test_workflows.py
+++ b/tardis/workflows/tests/test_workflows.py
@@ -163,7 +163,7 @@ def test_simple_tardis_workflow_against_test_tardis_full_regression(
         j_blue_estimators,
         expected_j_blue_estimators,
         rtol=1e-14,
-        atol=1e-14
+        atol=1e-14 # many values equal to zero
     )
 
     spectrum_real_packets_luminosity = (

--- a/tardis/workflows/tests/test_workflows.py
+++ b/tardis/workflows/tests/test_workflows.py
@@ -162,6 +162,8 @@ def test_simple_tardis_workflow_against_test_tardis_full_regression(
     np.testing.assert_allclose(
         j_blue_estimators,
         expected_j_blue_estimators,
+        rtol=1e-14,
+        atol=1e-14
     )
 
     spectrum_real_packets_luminosity = (
@@ -173,6 +175,8 @@ def test_simple_tardis_workflow_against_test_tardis_full_regression(
     assert_quantity_allclose(
         spectrum_real_packets_luminosity,
         expected_spectrum_real_packets_luminosity,
+        rtol=1e-14,
+        atol=0
     )
 
     spectrum_virtual_packets_luminosity = (
@@ -184,6 +188,8 @@ def test_simple_tardis_workflow_against_test_tardis_full_regression(
     assert_quantity_allclose(
         spectrum_virtual_packets_luminosity,
         expected_spectrum_virtual_packets_luminosity,
+        rtol=1e-14,
+        atol=0
     )
 
 

--- a/tardis/workflows/tests/test_workflows.py
+++ b/tardis/workflows/tests/test_workflows.py
@@ -176,7 +176,7 @@ def test_simple_tardis_workflow_against_test_tardis_full_regression(
         spectrum_real_packets_luminosity,
         expected_spectrum_real_packets_luminosity,
         rtol=1e-14,
-        atol=0
+        atol=None
     )
 
     spectrum_virtual_packets_luminosity = (
@@ -189,7 +189,7 @@ def test_simple_tardis_workflow_against_test_tardis_full_regression(
         spectrum_virtual_packets_luminosity,
         expected_spectrum_virtual_packets_luminosity,
         rtol=1e-14,
-        atol=0
+        atol=None
     )
 
 

